common/metrics/defs.go

```diff
@@ -689,6 +698,9 @@ const (
 	DynamicConfigScope
 
 	NumCommonScopes
+
+	// TieredStorageQueueProcessorScope is the scope used by all metric emitted by tiered storage queue processor
+	TieredStorageQueueProcessorScope
```

Comment:
This will break things. The NumCommonScopes is used as a counter below so the ids don't conflict.

Reply:
Good catch, moved right above NumCommonScopes.